src/util/getEventHandlers.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path')
+const glob = require('glob')
+
+const messageModule = require.resolve('~/src/messages')
+const messagesDir = messageModule.substring(0, messageModule.length - '/index.js'.length)
+
+const eventHandlers = {}
+let initialized = false
+
+module.exports = function getEventHandlers () {
+  if (!initialized) {
```

Comment:
This check might be overkill (I don't expect this function to ever be called twice).